### DISCUSSION TITLE
[Deprecation] Custom state pseudo classes from Chrome

### DISCUSSION THREAD

**Discussion by sgurin** [vimeo/player.js#1023] [Bug reports]
Hi. 

Chrome console show lots of errors on https://player.vimeo.com/api/demo example. Is that ok?

![image](https://github.com/vimeo/player.js/assets/25930200/45abce76-c48b-47fa-a9bb-cd45e250e408)

**Comment by bdougherty** [comment]
Yes, these are fine and can be ignored.
